src/builtins/Transformers.jl

```diff
@@ -177,6 +179,91 @@ MLJBase.docstring(::Type{<:FeatureSelector})      = "Filter features (columns) o
 MLJBase.load_path(::Type{<:FeatureSelector})      = "MLJModels.FeatureSelector"
 
 
+
+#### UNIVARIATE Discretizer ####
+"""
+UnivariateDiscretizer(n_classes=512)
+Returns a `MLJModels` for discretizing vectors of `Real`
+eltype, where `n_classes` describes the resolution of the
```

Comment:
- `MLJModel` not `MLJModels`.

- Can we instead make this "for discretising any `Continuous` vector `v` (`scitype(v) <: AbstractVector{Continuous}`)" ?